### PR TITLE
feat(theme): New theme poshmon

### PR DESCRIPTION
**Pull request by P1P0S**
### Prerequisites

- [x] I have read and understood the [contributing guide][CONTRIBUTING.md]
- [x] The commit message follows the [conventional commits][cc] guidelines

### Description

Features:  Pokemon based theme for Oh My Posh

poshmon

### CODE REVIEW

**Review by the-eduardo — APPROVED** [review]
New Theme: Poshmon

**Review by the-eduardo — CHANGES_REQUESTED** [review]
image

image

Need to check this visual glitch

File: themes/poshmon.omp.json [hunk 1]
```diff
@@ -0,0 +1,126 @@
...
+    {
+      "alignment": "left",
+      "segments": [
+        {
+          "background": "#2e9599",
+          "foreground": "#ffffff",
+          "leading_diamond": "\ue0b6",
+          "powerline_symbol": "\ue0b0",
+          "style": "diamond",
+          "template": "\uf91c {{ .HostName }} ",
```
Comment (the-eduardo) [inline themes/poshmon.omp.json]: Add visual fix changing line 13 to: `"template": "\uf91c {{ .HostName }}\u2800",`

**Review by the-eduardo — CHANGES_REQUESTED** [review]

File: themes/poshmon.omp.json (hunk 1 above)
Comment (the-eduardo) [reply themes/poshmon.omp.json]: image

**Review by P1P0S — COMMENTED** [review]
Thanks

**Review by the-eduardo — APPROVED** [review]

File: themes/poshmon.omp.json (hunk 1 above)
Comment (JanDeDobbeleer) [reply themes/poshmon.omp.json]: @the-eduardo this fix is to fix a bug in Windows Terminal and that's going to provide cross platform inconsistency (as it prints characters elsewhere).

**Pull request closed without merging**